generate_deb.sh

```diff
@@ -43,6 +43,11 @@ if [ -e ${build_dir}/packaging/Dockerfile.dep ]; then
 	cp ${build_dir}/packaging/Dockerfile.dep .
 	sed -i '/^### INCLUDE_DEPENDENCIES/ r ./Dockerfile.dep' ./Dockerfile
 fi
+if [ ! -z "${COVERITY_URL}" ]; then
+  sed -i "s|COVERITY_URL|$COVERITY_URL|g" ./packaging/common/Dockerfile.coverity
+	sed -i '/^### INCLUDE_DEPENDENCIES/ r ./packaging/common/Dockerfile.coverity' ./Dockerfile
```

Comment:
Does this work in case the Dockerfile.dep (above) has already replaced the "### INCLUDE_DEPENDENCIES" string in Dockerfile?